doc/installing.rst

```diff
@@ -115,6 +98,38 @@ running, e.g.:
     marks may not be correct, leading to errors when they are directly pasted
     into a terminal window.
 
+.. _optional-dependencies:
+
+Optional Dependencies
+^^^^^^^^^^^^^^^^^^^^^
+
+PypeIt has a number of optional dependencies that improve and/or expand functionality.
+
+    - If you are generating datacubes (and performing an astrometric
+      correction), you will also need the `scikit-image`_ package. It can be
+      installed by including it in the optional dependencies, e.g.:
+
+      .. code-block:: console
+
+        pip install "pypeit[pyside2,scikit-image]"
+
+    - PypeIt can use the `bottleneck`_ package to speed up a few calculations,
```

Comment:
`bottleneck` is now part of `install_requires` and thus installed by default. i forgot to edit this part of the docs when i made that change in my recent PR...

Reply:
Good point!  I removed that paragraph.